app/Observers/EntryObserver.php

```diff
@@ -14,10 +14,12 @@ class EntryObserver
      */
     public function created(Entry $entry)
     {
-        $character = $entry->character;
-        $character->level += $entry->levels;
+        if ($entry->type != Entry::TYPE_DM) {
```

Comment:
I think we should have a better check for the observer functions. Once we have entries attached to a character, we might want levels ups to be tracked if the player chooses to level up using that entry.

Reply:
If an entry specifies a level, the character should gain a level from that entry. That's currently how its is implemented from what I can see.

Reply:
I suppose it is possible that a user could create an entry that is both attached to a character as well as being of type 'dm' at creation. In this case the check here would be preventing such an entry from levelling that character. That's a change I can look into tomorrow.